omniport/core/open_auth/serializers/application.py

```diff
@@ -135,10 +137,26 @@ def validate_redirect_uris(self, value):
         """
 
         validator = URLValidator()
+        schemes = ['http', 'https', 'ftp', 'ftps']
+        regex = re.compile(
+            r'(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)|'  # domain...
+            r'localhost|'  # localhost...
+            r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'  # ...or ip
+            r'(?::\d+)?'  # optional port
+            r'(?:/?|[/?]\S+)$', re.IGNORECASE)
 
         redirect_urls = value.split(' ')
         for url in redirect_urls:
-            validator(url)
+            if '://' not in url:
```

Comment:
Why do we still have to do this? Doesn't the RegEx you wrote check for this?

Reply:
Actually, I wrote the custom regex for validating the URLs without any scheme, rest it is able to validate with any scheme we provide using URLValidator.